bouncer/util.py

```diff
@@ -70,4 +101,35 @@ def parse_document(document):
             _("The annotation has an invalid document URI"),
             "uri_not_a_string")
 
-    return (annotation_id, document_uri)
+    return {
+            "annotation_id": annotation_id,
+            "document_uri": document_uri,
+            "can_reveal_metadata": can_reveal_metadata,
```

Comment:
No need to change, but my personal preference is to keep variable names short and descriptive. Something like `show_metadata` should be enough and you can save yourself from typing one more world followed by an underscore.